how-defi-kingdoms-works/the-gardens.md

```diff
@@ -40,30 +40,30 @@ Please note that block speed fluctuates and should be viewed on an average (i.e.
 
 **Starting the first Epoch, 5% of any rewards claimed from the Gardens were unlocked and immediately accessible and 95% of rewards claimed from the Gardens were locked. The percentage of rewards CLAIMED from the Garden that are unlocked will increase by 2% every Epoch until after Epoch 51 where any rewards claimed thereafter will be 100% unlocked. This 2% increase of unlocked rewards per Epoch applies only to unclaimed rewards (i.e. once claimed, locked rewards do not benefit from this increasing unlock rate per Epoch and will remain locked until after Epoch 51 when they will start to unlock)** 
 
 So, for example, Garden staking rewards claimed during Epoch 1 will be 5% unlocked, 95% locked; during Epoch 2 will be 7% unlocked, 93% locked; during Epoch 3 will be 9% unlocked, 91% locked, and so on, until after Epoch 51, when no more locking of staked rewards will occur for newly claimed Garden rewards.
 
 ### Locking Model - Simplified Examples
 
-Below are two simplified examples that show how the locking model works. For the sake of simplicity, **the examples assume no reinvestment** of unlocked claimed rewards into additional LP positions and that a player earned 100 Jewel in Garden rewards during Epoch 1, 50 Jewel in Epoch 2 and so on to reflect the reduced Jewel emission per Epoch. In both scenarios, 301.6 Jewel is earned, but the amount unlocked versus locked is different. This locking model provides players the opportunity to choose a strategy that suits their style and risk appetite.
+Below are two simplified examples that show how the locking model works. For the sake of simplicity, **the examples assume no reinvestment** of unlocked claimed rewards into additional LP positions and that a player earned 100 JEWEL in Garden rewards during Epoch 1, 50 JEWEL in Epoch 2 and so on to reflect the reduced JEWEL emission per Epoch. In both scenarios, 301.6 JEWEL is earned, but the amount unlocked versus locked is different. This locking model provides players the opportunity to choose a strategy that suits their style and risk appetite.
 
 **Illustrative Example A**: Rewards earned during each Epoch are claimed during each Epoch (and not reinvested).
 
 * _10 Epoch Summary_:
-  * Total unlocked Jewel: **29.8** Jewel
-  * Total locked Jewel: **271.8** Jewel (inaccessible until after Epoch 51 unless unlocked earlier via Heroes)
-  * Total Jewel: **301.6** Jewel
+  * Total unlocked JEWEL: **29.8** JEWEL
+  * Total locked JEWEL: **271.8** JEWEL (inaccessible until after Epoch 51 unless unlocked earlier via Heroes)
+  * Total rewards: **301.6** JEWEL
```

Comment:
changed `JEWEL `to `rewards` to match the other example, but either would work fine